src/gluonts/dataset/split.py

```diff
@@ -15,36 +15,65 @@
 Train/test splitter
 ~~~~~~~~~~~~~~~~~~~
 
+.. testsetup:: *
```

Comment:
What is this?

Reply:
It's to add doctest snippets that run, but are not displayed, see: https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testsetup

For example for imports or some other setup that is used by following doctest snippets.